src/definitions/modules/dropdown.js

```diff
@@ -795,7 +795,7 @@ $.fn.dropdown = function(parameters) {
                 }
                 module.remove.message();
                 module.setup.menu({
-                  values: values
+                  [fields.values]: values
```

Comment:
As @ColinFrick pointed out, this won't work in IE11 (which we unfortunately have to support (at least it should not break existing code))
You can fix it by using a helper object like the `select` you used below

```javascript
var menuConfig = {};
menuConfig[fields.values] = values;
module.setup.menu(menuConfig);
```